testing/testcompat/v232/com/google/idea/sdkcompat/indexing/ProjectIndexingHistoryWrapper.java

```diff
@@ -79,7 +79,6 @@ public static ProjectIndexingHistoryWrapper create(
                   /* totalIndexingVisibleTime= */ new JsonDuration(
                       /* nano= */ providerIndexingTime.toNanos()),
                   /* contentLoadingVisibleTime= */ new JsonDuration(/* nano= */ 0),
-		  /* readActionWaitingVisibleTime= */ new JsonDuration(/* nano= */0),
```

Comment:
A recent change in this constructor got reverted
https://github.com/JetBrains/intellij-community/commit/d481c0d67e1fecdaf29f1b9f8fe99c6df4a62f61